packages/models/src/resource-binding/utils.ts

```diff
@@ -1,6 +1,17 @@
 import { OutPointString } from '../store'
-import { OutPoint } from '@ckb-lumos/base'
+import { Header, OutPoint } from '@ckb-lumos/base'
+import { Manager } from './manager'
+import { ActorReference } from '../actor'
+import { ProviderKey } from '../utils'
+import { ChainSource } from '@ckb-js/kuai-io/lib/types'
+import { Listener } from '@ckb-js/kuai-io'
```

Comment:
Most are type declarations and could be prepended by `type` keyword for explicitness.